modules/porous_flow/test/tests/sinks/s11.i

```diff
@@ -0,0 +1,135 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 5
+  ny = 5
+  nz = 5
+  xmin = 0
+  xmax = 10
+  ymin = 0
+  ymax = 10
+  zmin = 0
+  zmax = 10
+[]
+
+[GlobalParams]
+  PorousFlowDictator = dictator
+[]
+
+[UserObjects]
+  [./dictator]
+    type = PorousFlowDictator
+    porous_flow_vars = 'pp temp'
+    number_fluid_phases = 1
+    number_fluid_components = 1
+  [../]
+  [./pc]
+    type = PorousFlowCapillaryPressureVG
+    m = 0.5
+    alpha = 1
+  [../]
+[]
+
+[Variables]
+  [./pp]
+    initial_condition = 1
+  [../]
+  [./temp]
+    initial_condition = 2
+  [../]
+[]
+
+[Kernels]
+  [./mass0]
+    type = PorousFlowMassTimeDerivative
+    fluid_component = 0
+    variable = pp
+  [../]
+
+  [./heat_conduction]
+    type = TimeDerivative
+    variable = temp
+  [../]
+[]
+
+[Modules]
+  [./FluidProperties]
+    [./simple_fluid]
+      type = SimpleFluidProperties
+      bulk_modulus = 1
+      density0 = 10
+      thermal_expansion = 0
+      viscosity = 11
+    [../]
+  [../]
+[]
+
+[Materials]
+  [./ppss]
+    type = PorousFlow1PhaseP
+    porepressure = pp
+    capillary_pressure = pc
+  [../]
```

Comment:
Can use PorousFlow1PhaseFullySaturated instead - i think i mentioned this before